ironmon_tracker/Utils.lua

```diff
@@ -184,7 +184,11 @@ function Utils.getDetailedEvolutionsInfo(evoMethod)
 	end
 
 	if evoMethod == PokemonData.Evolutions.FRIEND then
-		return { "220 Friendship" }
+		if Program.friendshipRequired ~= nil and Program.friendshipRequired > 1 then
+			return { Program.friendshipRequired .. " Friendship" }
+		else
+			return { "220 Friendship" }
+		end
```

Comment:
This else block here wouldn't ever activate since `Program.friendshipRequired` only gets changed in `Program.Initialize` if the read value is between 1 and 220, and is initialised to 220 in the definition of the Program variables

Not an issue, and still a potential safety net if something goes horribly wrong i suppose, but just noting that this if else is made redundant by previous safety checks

Reply:
Aye, thanks for pointing this out. I did realize this as I was writing it, but decided to leave it in just as a Default value in case someone else changed something, or something weird happened with setting the friendship value.